### PR TITLE
adding ops task to rundeck

### PR DESCRIPTION
**Pull request by matthewfl**
adds support for fabric tasks to our rundeck

ping @balanced-cookbooks/chefs

Also the permissions on this repo appear to be setup wrong

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -24,3 +24,5 @@ group(:test) do
   cookbook 'apt'
   cookbook 'balanced-rundeck_test', path: 'test/cookbooks/balanced-rundeck_test'
 end
+
+cookbook 'rundeck-fabric', path: '../rundeck-fabric'
```
Comment (mjallday) [inline Berksfile]: is this not published yet?

Comment (matthewfl) [inline Berksfile]: https://github.com/balanced-cookbooks/rundeck-fabric/pull/6

File: recipes/ops.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+rundeck_fabric_project 'ops' do
```
Comment (coderanger) [inline recipes/ops.rb:19]: Needs license header.

File: metadata.rb [hunk 3]
```diff
@@ -29,3 +29,4 @@
 depends 'balanced-user', '~> 1.0.14'
 depends 'rundeck', '~> 99.1.0'
 depends 'partial_search', '~> 1.0.7'
+depends 'rundeck-fabric', '~> 1.0.0'
```
Comment (coderanger) [inline metadata.rb:32]: Should be `~> 1.0` for semver matching.

**Pull request merged**